endpoints.tf

```diff
@@ -0,0 +1,15 @@
+data "aws_vpc_endpoint_service" "transferserver" {
+  count   = var.transferserver_endpoint != null ? 1 : 0
+  service = "transfer.server"
+}
+
+resource "aws_vpc_endpoint" "transferserver" {
+  count               = var.transferserver_endpoint != null ? 1 : 0
+  private_dns_enabled = var.transferserver_endpoint.private_dns_enabled
+  security_group_ids  = var.transferserver_endpoint.security_group_ids
+  service_name        = data.aws_vpc_endpoint_service.transferserver[0].service_name
+  subnet_ids          = var.transferserver_endpoint.subnet_ids
+  tags                = merge(var.tags, { "Name" = "${var.stack}-transferserver_endpoint" })
```

Comment:
I suggest to use either `-` or `_` in the name, but not both.

Reply:
Adding to this, can we drop the `-endpoint` suffix please? We know it's an endpoint by the type of resource.  

And following on from previous comment this should look like `"${var.stack}-transfer-server"`, I'm not a fan of concatenating words when already using kebab- or snake-case in a string.

Reply:
Ok